packages/uhk-web/src/app/components/agent/about/about.component.ts

```diff
@@ -1,8 +1,16 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Observable } from 'rxjs/Observable';
+
 import { Constants } from 'uhk-common';
 
 import { getVersions } from '../../../util';
 
+import { AppState, contributors } from './../../../store';
```

Comment:
the `./` does not need in the imports. pls remove it every where

Reply:
Vscode auto imports. Gonna create a rule for this...